src/main/java/no/nav/familie/integrasjoner/journalpost/BaksTilgangsstyrtJournalpostService.kt

```diff
@@ -0,0 +1,58 @@
+package no.nav.familie.integrasjoner.journalpost
+
+import no.nav.familie.integrasjoner.baks.søknad.BaksVersjonertSøknadService
+import no.nav.familie.integrasjoner.tilgangskontroll.TilgangskontrollService
+import no.nav.familie.kontrakter.felles.Tema
+import no.nav.familie.kontrakter.felles.journalpost.Journalpost
+import no.nav.familie.kontrakter.felles.journalpost.TilgangsstyrtJournalpost
+import no.nav.familie.kontrakter.felles.søknad.MissingVersionException
+import no.nav.familie.kontrakter.felles.søknad.UnsupportedVersionException
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+
+@Service
+class BaksTilgangsstyrtJournalpostService(
+    private val baksVersjonertSøknadService: BaksVersjonertSøknadService,
+    private val tilgangskontrollService: TilgangskontrollService,
+) {
+    fun tilTilgangstyrteJournalposter(journalposter: List<Journalpost>): List<TilgangsstyrtJournalpost> =
```

Comment:
Kanskje det blir litt mer tydeligere med å skrive `map` først i metoden?

```suggestion
    fun mapTilTilgangstyrteJournalposter(journalposter: List<Journalpost>): List<TilgangsstyrtJournalpost> =
```